src/libs/actions/IOU.ts

```diff
@@ -7067,7 +7069,7 @@ function canApproveIOU(iouReport: OnyxTypes.OnyxInputOrEntry<OnyxTypes.Report>,
         }
     }
 
-    return isCurrentUserManager && !isOpenExpenseReport && !isApproved && !iouSettled && !isArchivedReport && !isTransactionBeingScanned;
+    return isCurrentUserManager && !isOpenExpenseReport && !isApproved && !iouSettled && !isArchivedReport && !isArchivedExpenseReport && !isTransactionBeingScanned;
```

Comment:
I don't think we need `isArchivedReport` here either but please let me know if you see something I'm missing.